### PR TITLE
Added Lords of Vegas

### PR DESCRIPTION
**Pull request by chinmay7016**
#269 
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by chinmay7016** [comment]
@edm00se if you like this changes pls merge this pr

File: readme.md [hunk 1]
```diff
@@ -2517,6 +2517,29 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+
+The game board is broken into 6 different areas, each consisting of a number of empty 'lots'. Players build lots by paying money and placing a die of the value matching the one shown on the lot's space onto the lot, along with a casino tile of one of 7 colors. Adjoining lots of the same color are considered a single casino. The casino's boss is the player whose die value is higher than any other in the casino. On each players turn, players turn over a new card representing a new lot they get. The card also is one of the casino colors. Any built casinos of the matching color will score both money and points. Money is earned for each lot in the casino, where each lot may be owned by a different player. Points go only to the casino's owner. Players can expand their casinos; try to take over casinos owned by other players; make deals to trade lots, casinos and money; or gamble in opponents' casinos to make more money. Ultimately, though, only points matter, and that means making yourself boss of the biggest casinos.
```
Comment (edm00se) [inline readme.md:2524]: Please update the formatting here as well.

File: readme.md [hunk 2]
```diff
@@ -2517,6 +2517,29 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+
+Lords of Vegas contains:
```
Comment (edm00se) [inline readme.md:2526]: This list can be removed.

**Review by edm00se — CHANGES_REQUESTED** [review]
Looks like the description block formatting is needed here as well. Also please remove the ~”this game has…” list. Everything else looks good.

**Comment by chinmay7016** [comment]
@edm00se  sir pls review changes and merge it pls sir.

**Comment by chinmay7016** [comment]
@edm00se sir pls review changes and merge it pls sir.

**Review by edm00se — APPROVED** [review]

**Pull request merged**